build/Starboard/dll/Starboard.def

```diff
@@ -177,6 +175,7 @@ LIBRARY Starboard
         EbrEventTimedWait
         EbrEventTimedMultipleWait
         EbrEventDestroy
+        _EbrGetBaseWriteablePath
 
         ; REMOVE / CLEANUP C++ EXPORTS
         ?format@string@woc@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PBDZZ
```

Comment:
probably? Despite outward appearances, my goal is not to fix everything potentially wrong with this project.